src/js/app.js

```diff
@@ -41,32 +41,22 @@ import RatingPointCard from './components/point-card/rating-point-card';
 // Right now, it's hash based.
 import history from './history';
 
-// Builds a redux store given optional middleware.
-// TODO: refactor out this dependency once we only depend on redux-thunk.
-import StoreBuilder from './store';
-
-// Allow direct access to the local PouchDB instance and import the gateway
-// singleton to make working with the database easier.
-import database from './database';
-import gateway from './gateway';
```

Comment:
Don't need a special gateway class anymore, we're using backbone-pouch in the btc-models.